hack/test/unit

```diff
@@ -18,16 +18,32 @@ TESTDIRS="${TESTDIRS:-./...}"
 exclude_paths='/vendor/|/integration'
 pkg_list=$(go list $TESTDIRS | grep -vE "($exclude_paths)")
 
-echo "${pkg_list}" | grep --fixed-strings "libnetwork/drivers/bridge" \
+base_pkg_list=$(echo "${pkg_list}" | grep --fixed-strings -v "/libnetwork" || :)
+libnetwork_pkg_list=$(echo "${pkg_list}" | grep --fixed-strings "/libnetwork" || :)
+
+echo "${libnetwork_pkg_list}" | grep --fixed-strings "libnetwork/drivers/bridge" \
 	&& if ! type docker-proxy; then
 		hack/make.sh binary-proxy install-proxy
 	fi
 
 mkdir -p bundles
-gotestsum --format=standard-quiet --jsonfile=bundles/go-test-report.json --junitfile=bundles/junit-report.xml -- \
-	"${BUILDFLAGS[@]}" \
-	-cover \
-	-coverprofile=bundles/profile.out \
-	-covermode=atomic \
-	${TESTFLAGS} \
-	${pkg_list}
+
+if [ "${base_pkg_list}" ]; then
+	gotestsum --format=standard-quiet --jsonfile=bundles/go-test-report.json --junitfile=bundles/junit-report.xml -- \
+		"${BUILDFLAGS[@]}" \
+		-cover \
+		-coverprofile=bundles/profile.out \
+		-covermode=atomic \
+		${TESTFLAGS} \
+		${base_pkg_list}
+fi
+if [ "${libnetwork_pkg_list}" ]; then
+	gotestsum --format=standard-quiet --jsonfile=bundles/go-test-report-libnetwork.json --junitfile=bundles/junit-report-libnetwork.xml -- \
+		"${BUILDFLAGS[@]}" \
+		-cover \
+		-coverprofile=bundles/profile-libnetwork.out \
+		-covermode=atomic \
+		-p=1 \
```

Comment:
perhaps would be good to add a comment in this branch/block to describe what it's different from the other block (`-p 1`)

Reply:
And why :)

Reply:
did it in https://github.com/moby/moby/pull/42594/commits/1135681ccb31e6e106b2141f8570d3bf06c78ea3